tests/FSharp.Compiler.ComponentTests/Interop/SimpleInteropTests.fs

```diff
@@ -33,7 +33,7 @@ let a = AMaker.makeA()
         |> shouldSucceed
 
 
-    [<Fact>]
+    // TODO: This is broken RN, since netcoreapp30 is used for C# and 3.1 for F#, should be fixed as part of https://github.com/dotnet/fsharp/issues/9740
```

Comment:
A good practice here is rather than commenting out the `Fact` attribute, add `Skip="This is broken RN, since netcoreapp30 is used for C# and 3.1 for F#, should be fixed as part of https://github.com/dotnet/fsharp/issues/9740"`. This way it shows up in the skipped tests lists and remains more visible.

Reply:
Yeah, good point, will fix that.